pkg/cmd/roachtest/roachtestutil/mixedversion/helper.go

```diff
@@ -297,6 +298,17 @@ func (h *Helper) loggerFor(name string) (*logger.Logger, error) {
 	return prefixedLogger(h.runner.logger, fileName)
 }
 
+// IsSkipVersionUpgrade returns if the current upgrade is a skip
+// version upgrade. This can be used to gate steps that are
+// incompatible with version skipping, i.e. the schema change workload.
+//
+// N.B. Calling this in a startup step may lead to unexpected results,
+// as there is no upgrade going on.
+func (h *Helper) IsSkipVersionUpgrade() bool {
+	numReleases, _ := release.MajorReleasesBetween(&h.testContext.FromVersion().Version, &h.testContext.ToVersion().Version)
```

Comment:
We can ignore the error because both `FromVersion` and `ToVersion` are validated elsewhere?

Reply:
Stealing Renato's comment (and also adding it to the code above):

```
	// N.B.: it is safe to ignore the error here as we should always be
	// able to determine the number of releases between two binaries
	// used when upgrading: we keep release data starting from 21.2.
```

From my understanding that reasoning would apply here as well.